torch_spyre/_inductor/codegen/superdsc.py

```diff
@@ -94,6 +95,25 @@ def generate_sdsc(pointers, *, op, dimensions, inputs, outputs, reduction, **kwa
         else:
             # Non-stick transpose currently unsupported
             raise Unsupported("Transposition not changing the stick dimension")
+    if op == TRANSPOSE_OP and len(dimensions) == 4:
+        transposed_dims = [
+            dim % len(dimensions) for dim in kwargs["op_info"]["transposed_dims"]
+        ]
+        is_supported = (0 in transposed_dims) and 3 in transposed_dims
```

Comment:
can you add a comment here explaining the conditions for the 4D transpose to be supported?